product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/18_dbms_xmldom/getnodename.mdx

```diff
@@ -12,11 +12,11 @@ GETNODENAME(n DOMNODE) RETURN VARCHAR2
 
 `n`
 
-`DOMNode` value to be provided.
+`DOMNode` value to provide.
```

Comment:
Since this page related to GETNODENAME, I assume what this does is to fetch/display the value of a DOMNode?
@nidhibhammar @sravanvelag 
```suggestion
`DOMNode` value you want to obtain
```

Reply:
Got it, let me know what you think about the previous suggestion and we'll fix this accordingly.